pkg/operator/staticpod/controller/installer/installer_controller_test.go

```diff
@@ -2084,22 +2085,123 @@ func TestInstallerController_manageInstallationPods(t *testing.T) {
 		want    bool
 		wantErr bool
 	}{
-		// TODO: Add test cases.
+		{
+			name: "if the revision shouldn't install and a new revision is to be rolled out, requeue the new installer pod",
+			fields: fields{
+				targetNamespace: "test-namespace",
+				staticPodName:   "test-pod",
+				command:         []string{},
+				kubeClient:      fake.NewSimpleClientset(),
+				eventRecorder:   eventstesting.NewTestingEventRecorder(t),
+				shouldRevisionInstall: func() (bool, error) {
+					return false, fmt.Errorf("revision shouldn't be installed")
```

Comment:
ah okay, now I get it. If you want to use an error for control flow, that's fine. But then we don't really need the boolean to denote whether to skip it or not. 

I (personally) would use the error for real errors though.